src/utils/defs.js

```diff
@@ -0,0 +1,2 @@
+export const ACCEPTED = "accepted";
+export const DECLINED = "declined";
```

Comment:
you called I `defs`, as the one who despises abbreviations 🤣 ? ok I would expand it cuz even I didn't got right away, or maybe name it `const`.

Reply:
`src/utils/constants.js` or `src/utils/definitions.js`?

I don't really like `const` or `constants`, but I could live with it.

Reply:
`src/utils/definitions.js` 👍